src/EnablerExtension.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace SilverStripe\LoginForms;
+
+use SilverStripe\View\SSViewer;
+use SilverStripe\ORM\DataExtension;
+use SilverStripe\Security\Security;
+use SilverStripe\Core\Config\Config;
+
+/**
+ * Applies to the {@see Security} controller in order to detect requests for actions related to
+ * the log in process or other such credential management (such as the forgot password flow).
+ * This is in order to replace the set {@see SSViewer} theme list with a conrolled set in order
+ * to always show a consistent interface that relates more to the CMS than the website it is
+ * loading on.
+ * Particular actions can be set to be ignored by inlcuding them in the `excluded_actions` list
+ * defined in yml _config for this class. By default all allowed actions on the Security controller
+ * excepting `index`, `logout`, and `ping` will have the new theme set applied.
```

Comment:
See previous comment about logout. Do you think it would be easier not to have whitelist/blacklists for specific actions, and simply apply the whole theme to the CMS instead? You could then remove this extension

Reply:
This is not in the CMS.
I considered applying simply to `allowed_actions` but added the blacklist as it is possible to extend the Security controller (as I have) to add actions - which in turn may delegate to other controllers. The reason for the black list is to avoid irreparably negating any theme a delegated control flow may rely on. It's an exceedingly simple bit of functionality that adds the ability to allow for unforeseen customisations.

Reply:
>This is not in the CMS.

It basically is, that's the point of this theme - to remove the "login forms look different on each site unless they design their own theme templates".

Justification seems OK, can you add some unit tests please?